config/sync/webform.webform.descriptive_metadata.yml

```diff
@@ -14,7 +14,298 @@ id: descriptive_metadata
+elements: |
+  metadata:
+    '#type': webform_wizard_page
+    '#title': Metadata
+    '#access_create_roles':
+      - administrator
+      - metadata_pro
+      - curator
+    '#access_update_roles':
+      - authenticated
+    '#access_view_roles':
+      - authenticated
+    '#next_button_label': 'Move on to next step'
+    label:
+      '#type': textfield
+      '#title': Title
+      '#help': 'Primary title given to described resource'
+      '#minlength': '3'
+      '#maxlength': '256'
+      '#placeholder': 'Title of this Object'
+      '#required': true
+      '#label_attributes':
+        class:
+          - custom-form-input-heading
+      '#format_items': comma
+      '#access_create_roles':
+        - administrator
+      '#access_update_roles':
+        - administrator
+    type:
+      '#type': select
+      '#title': 'Media type'
+      '#options': schema_org_creative_works
+      '#required': true
+      '#label_attributes':
+        class:
+          - custom-form-input-heading
+    description:
+      '#type': textarea
+      '#title': Description
+      '#title_display': before
+      '#rows': '4'
+      '#autocomplete': 'off'
+      '#required': true
+      '#required_error': 'Please describe this object. It''s super needed.'
+      '#label_attributes':
+        class:
+          - custom-form-input-heading
+      '#format_items': comma
+    geographic_location:
+      '#type': webform_metadata_nominatim
+      '#title': 'Location of asset'
+      '#description': 'Example: <em>Central Park, Manhattan, New York County, New York, United States of America</em>'
+      '#help_title': Help
+      '#help': |
+        The location is meant to describe where the panorama was made with as much information as possible.<br />
```

Comment:
A few replacements here to make this more general:
Title should be  '#title': 'Location related to this Digital Object'
Replace "...describe where the panorama was made" with something like "describe a geographic location closely related to this Digital Object (Origin, Where it was created, etc), e.g as a way of contextualizing this on a map"